catalyst-gateway/bin/src/db/index/block/cip36/insert_cip36.rs

```diff
@@ -0,0 +1,83 @@
+//! Insert CIP36 Registration Query
+
+use std::sync::Arc;
+
+use cardano_chain_follower::Metadata::cip36::{Cip36, VotingPubKey};
+use scylla::{frame::value::MaybeUnset, SerializeRow, Session};
+use tracing::error;
+
+use crate::{
+    db::index::queries::{PreparedQueries, SizedBatch},
+    settings::CassandraEnvVars,
+};
+
+/// Index Registration by Stake Address
+const INSERT_CIP36_REGISTRATION_QUERY: &str = include_str!("./cql/insert_cip36.cql");
+
+/// Insert CIP-36 Registration Query Parameters
+#[derive(SerializeRow, Clone)]
+pub(super) struct Params {
+    /// Stake key hash
+    stake_address: Vec<u8>,
+    /// Nonce value after normalization.
+    nonce: num_bigint::BigInt,
+    /// Slot Number the cert is in.
+    slot_no: num_bigint::BigInt,
+    /// Transaction Index.
+    txn: i16,
+    /// Voting Public Key
```

Comment:
```suggestion
    /// Voting key hash
```

Reply:
It is the Voting Public key,  we never hash the voting key.